src/CoordinateReferenceSystem/CoordinateReferenceSystem.php

```diff
@@ -75,16 +75,11 @@ final public static function jsonUnserialize($json): self
 
         $type = (string) $json['type'];
         $properties = $json['properties'];
-
-        switch ($type) {
-            case 'link':
-                return Linked::jsonUnserializeFromProperties($properties);
-
-            case 'name':
-                return Named::jsonUnserializeFromProperties($properties);
-        }
-
-        throw UnserializationException::unsupportedType('CRS', $type);
+        return match ($type) {
```

Comment:
```suggestion

        return match ($type) {
```

Extra line before the `return` statement for readability.